pvlib/pvsystem.py

```diff
@@ -758,21 +758,32 @@ def i_from_v(self, resistance_shunt, resistance_series, nNsVth, voltage,
 
     # inverter now specified by self.inverter_parameters
     def snlinverter(self, v_dc, p_dc):
-        """Uses :func:`snlinverter` to calculate AC power based on
+        """Uses :pyfunc:`pvlib.inverter.sandia` to calculate AC power based on
         ``self.inverter_parameters`` and the input parameters.
 
         Parameters
         ----------
-        See pvsystem.snlinverter for details
+        See pvlib.inverter.sandia for details
 
         Returns
         -------
-        See pvsystem.snlinverter for details
+        See pvlib.inverter.sandia for details
         """
-        return snlinverter(v_dc, p_dc, self.inverter_parameters)
+        return inverter.sandia(v_dc, p_dc, self.inverter_parameters)
 
     def adrinverter(self, v_dc, p_dc):
-        return adrinverter(v_dc, p_dc, self.inverter_parameters)
+        """Uses :pyfunc:`pvlib.inverter.adr` to calculate AC power based on
```

Comment:
```suggestion
        """Uses :py:func:`pvlib.inverter.adr` to calculate AC power based on
```

Reply:
@kanderso-nrel is there any reason we don't use the full path when a function is within the module?  E.g., for a docstring within `pvsystem.py`, :py:func:`pvlib.pvsystem.pvwatts_ac` instead of :py:func:`pvwatts_ac`

Reply:
As far as I know, it's just a style preference for whether RTD will display `pvlib.pvsystem.pvwatts_ac` or just `pvwatts_ac` in the link text.  I don't think pvlib consistently follows one style or the other currently.  The only reason I can think of to not include the full path is just to reduce clutter -- seems silly to have a bunch of `pvlib.` prefixes everywhere if the bare function names are sufficient.  But sometimes just the bare function name alone isn't very informative.  

It is possible to display custom text in these links:  ```:py:func:`title <target>` ```  ([docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#xref-syntax)).  Not saying we should be using it, it's just another option.

Reply:
My (not strong) preference would be to display full paths, although its more text there's no question where the function is found in the library. I'm sure that part of the reason that there's no consistent use is my lack of understanding of these markups.

Reply:
In `pvsystem.py` I've put full paths in where `:py:func:` is used.